content/documents/message-senders.mdx

```diff
@@ -237,51 +271,51 @@ If you're using our Mandrill/Mailchimp integration and providing your own custom
           <tbody>
             <tr>
               <td>cost</td>
-              <td>LineItem.LineTotal</td>
+              <td>Products[i].LineItem.LineTotal</td>
```

Comment:
also i believe this should be `LineItems[i].LineTotal` since the `Products` order event array would not have LineItem info.  same with other line item specific fields in this section

Reply:
updated